packages/shared/lib/contexts/governance/enums/index.ts

```diff
@@ -1,2 +1,3 @@
+export * from './proposal-order-option.enum'
 export * from './proposal-type.enum'
 export * from './proposal-status.enum'
```

Comment:
just sorting this by alphabetic order
```suggestion
export * from './proposal-status.enum'
export * from './proposal-type.enum'
```